modules/3d/src/pointcloud/coders_obj.cpp

```diff
@@ -0,0 +1,112 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "coders_obj.hpp"
+#include <fstream>
+#include <iostream>
```

Comment:
> #include <iostream>

avoid using of this header in the source code (it emits initialization code of streams).